[WIP] Augmentation model based on thesaurus of synonyms

SultanovAR opened the pull request.

vikmary commented:
```bash
python3 -m deeppavlov interact deeppavlov/configs/augmentation/thesaurus_aug.json -davlov interact deeppavlov/configs/augmentation/thesaurus_aug.json -d
```
выдает ошибку
```
ModuleNotFoundError: No module named 'pattern'
```

Возможно, поможет прописать requirements в конфиге

SultanovAR commented:
> ```shell
> python3 -m deeppavlov interact deeppavlov/configs/augmentation/thesaurus_aug.json -d
> ```
> 
> outputs the error:
> 
> ```
> ModuleNotFoundError: No module named 'pattern'
> ```
> 
> See the `requirements` section of config, it might help

fixed

SultanovAR commented:
> `File b'/home/vimary/.deeppavlov/downloads/ruthes_lite2/synonyms.csv' does not exist: b'/home/vimary/.deeppavlov/downloads/ruthes_lite2/synonyms.csv'`
> Have you included the following file in the download section?

Fixed, i have added download section in config file. now there are two config file (ru and en).
